python-sync-actions/src/user_functions.py

```diff
@@ -118,7 +118,7 @@ def strtotime(self, string, base_time=None):
 
         """
 
-        command = f"php -r 'echo strtotime(\"{string}\", {base_time or 'null'});'"
+        command = f"php -r 'echo strtotime(\"{string}\", {base_time or 'time()'});'"
```

Comment:
Tohle je OK, ale klidne bych tam narval Python current time => protoze bys na to mel napsat test v `test_funtions.py` tenhle test tam vubec nebyl a vzdycky se tam daval druhej parametr. Podle me by to spadlo protoze v Dockeru je nejaka jina verze PHP kde php -r 'echo strtotime("neco",null") vraci divnosti.
![image](https://github.com/user-attachments/assets/99d161c8-c3bc-4858-a496-cc242af3145b)

Reply:
:-D jo python time jsem tam přesně měl, ale pak jsem si všiml, že u date funkce tam máš ten php time(). Neva, dám tam python. Udělám to stejně i u date a doplním testy na date bez timestamp a strtotime bez base_time. Souhlasíš?

Reply:
Souhlas, Python time jsem navrhl protoze pak se ty testy udelaj snadno a bude ti stacit ten `freeze_time`

Reply:
změněno a testy přidány